DashAI/back/preprocessor/column_dropper_by_index.py

```diff
@@ -0,0 +1,55 @@
+from beartype import beartype
+from datasets import DatasetDict
+
+from DashAI.back.dataloaders.classes.dashai_dataset import DashAIDataset
+from DashAI.back.preprocessor.base_converter import BaseConverter
+
+
+class ColumnDropperByIndex(BaseConverter):
+    """Converter to drop columns from the dataset by column index"""
+
+    @beartype
+    def __init__(self, columns: tuple[int, int]):
```

Comment:
Why not call the parameter in a more descriptive way, such as `columns_index`?
```suggestion
    def __init__(self, columns_index: tuple[int, int]):
```